### PR TITLE
Add additional betfair enums

### PR DESCRIPTION
**Pull request by limx0**
This PR adds a bunch of missing enums and streaming_data_fields

### CODE REVIEW

**Comment by limx0** [comment]
Courtesy of @rozzac90

**Comment by limx0** [comment]
Not sure why this failed on 3.4.. will investigate

File: betfairlightweight/enums.py [hunk 1]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    Specify how to sort market data received from Betfair APING.
+    :var MINIMUM_TRADED: Minimum traded volume
+    :var MAXIMUM_TRADED: Maximum traded volume
+    :var MINIMUM_AVAILABLE: Minimum available to match
+    :var MAXIMUM_AVAILABLE: Maximum available to match
+    :var FIRST_TO_START: The closest markets based on their expected start time
+    :var LAST_TO_START: The most distant markets based on their expected start time
+    """
+    MinTraded = 'MINIMUM_TRADED'
+    maxTraded = 'MAXIMUM_TRADED'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: Case issue

File: betfairlightweight/enums.py [hunk 2]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    MinAvail = 'MINIMUM_AVAILABLE'
+    MaxAvail = 'MAXIMUM_AVAILABLE'
+    FirstStart = 'FIRST_TO_START'
+    LatestStart = 'LAST_TO_START'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: LastStart to match FirstStart?

File: betfairlightweight/enums.py [hunk 3]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+class OrderProjection(Enum):
+    """
+    Orders to include it market book data retrieved from exchange.
+    :var ALL: EXECUTABLE and EXECUTION_COMPLETE orders
+    :var EXECUTABLE: An order that has a remaining unmatched portion
+    :var EXECUTION_COMPLETE: An order that does not have any remaining unmatched portion
+    """
+    All = 'ALL'
+    Executable = 'EXECUTABLE'
+    Traded = 'EXECUTION_COMPLETE'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: ExecutableComplete to prevent confusion?

File: betfairlightweight/enums.py [hunk 4]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    """
+    Description supplied to tell exchange how to handle bet.
+    :var LAPSE: Lapse the order when the market is turned in-play
+    :var PERSIST: Persist the order to in-play. The bet will be place automatically into the in-play market at the start of the event.
+    :var MARKET_ON_CLOSE: Put the order into the auction (SP) at turn-in-play
+    """
+    Kill = 'LAPSE'
+    Keep = 'PERSIST'
+    ClosePrice = 'MARKET_ON_CLOSE'
+
```
Comment (liampauling) [inline betfairlightweight/enums.py]: Why not match? Kill may confuse people as it only 'kills' if the market goes inplay/suspends

File: betfairlightweight/enums.py [hunk 5]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    Status of orders sent, in processing or completed at the exchange.
+    :var PENDING: An asynchronous order is yet to be processed. Once the bet has been processed by the exchange (including waiting for any in-play delay),
+                  the result will be reported and available on the  Exchange Stream API and API NG. Not a valid search criteria on MarketFilter
+    :var EXECUTION_COMPLETE: An order that does not have any remaining unmatched portion.
+    :var EXECUTABLE: An order that has a remaining unmatched portion.
+    :var EXPIRED: The order is no longer available for execution due to its time in force constraint. Not a valid search criteria on MarketFilter.
+                  In the case of FILL_OR_KILL orders, this means the order has been killed because it could not be filled to your specifications.
+    """
+    Executed = 'EXECUTION_COMPLETE'
+    AtExchange = 'EXECUTABLE'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: Executable

File: betfairlightweight/enums.py [hunk 6]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    """
+    Status of orders sent, in processing or completed at the exchange.
+    :var PENDING: An asynchronous order is yet to be processed. Once the bet has been processed by the exchange (including waiting for any in-play delay),
+                  the result will be reported and available on the  Exchange Stream API and API NG. Not a valid search criteria on MarketFilter
+    :var EXECUTION_COMPLETE: An order that does not have any remaining unmatched portion.
+    :var EXECUTABLE: An order that has a remaining unmatched portion.
+    :var EXPIRED: The order is no longer available for execution due to its time in force constraint. Not a valid search criteria on MarketFilter.
+                  In the case of FILL_OR_KILL orders, this means the order has been killed because it could not be filled to your specifications.
+    """
+    Executed = 'EXECUTION_COMPLETE'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: ExecutionComplete

File: betfairlightweight/enums.py [hunk 7]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+
+
+class BetTargetType(Enum):
+    """
+    Specify a bet to be for a specific payout or profit.
+    :var BACKERS_PROFIT: The payout requested minus the calculated size at which this LimitOrder is to be placed
+    :var PAYOUT: The total payout requested on a LimitOrder
+    """
+    Payout = 'PAYOUT'
+    Profit = 'BACKERS_PROFIT'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: BackersProfit

File: betfairlightweight/enums.py [hunk 8]
```diff
@@ -236,3 +236,266 @@ class StreamingOrderType(Enum):
...
+    :var MAXIMUM_TRADED: Maximum traded volume
+    :var MINIMUM_AVAILABLE: Minimum available to match
+    :var MAXIMUM_AVAILABLE: Maximum available to match
+    :var FIRST_TO_START: The closest markets based on their expected start time
+    :var LAST_TO_START: The most distant markets based on their expected start time
+    """
+    MinTraded = 'MINIMUM_TRADED'
+    maxTraded = 'MAXIMUM_TRADED'
+    MinAvail = 'MINIMUM_AVAILABLE'
+    MaxAvail = 'MAXIMUM_AVAILABLE'
```
Comment (liampauling) [inline betfairlightweight/enums.py]: *Available on both

File: betfairlightweight/enums.py (hunk 4 above)
Comment (rozzac90) [inline betfairlightweight/enums.py]: Agreed kill is confusing, maybe not match though as it is also misleading lapse or cancel probably most logical.

File: betfairlightweight/enums.py (hunk 3 above)
Comment (rozzac90) [inline betfairlightweight/enums.py]: ExecutionComplete probably the most logical here.

**Pull request closed without merging**